src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java

```diff
@@ -500,6 +500,23 @@ public void jnlpWorkingDir() throws Exception {
         r.assertBuildStatusSuccess(r.waitForCompletion(b));
     }
 
+    @Issue("JENKINS-60517")
+    @Test
+    public void runInDynamicallyCreatedContainer() throws Exception {
+        List<PodTemplate> templates = cloud.getTemplates();
+        while (templates.isEmpty()) {
+            LOGGER.log(Level.INFO, "Waiting for template to be created");
+            templates = cloud.getTemplates();
+            Thread.sleep(1000);
+        }
+        assertFalse(templates.isEmpty());
+        PodTemplate template = templates.get(0);
+        assertEquals(Integer.MAX_VALUE, template.getInstanceCap());
+        r.assertBuildStatusSuccess(r.waitForCompletion(b));
```

Comment:
This is currently failing for me locally in Microk8s (noticed while testing #1083); the build fails in `docker run` with

```
docker: Error response from daemon: cgroups: cgroup mountpoint does not exist: unknown.
```

Perhaps because Microk8s is now using cgroup v2, and `docker:19-dind` is assuming v1? Need to figure out how to `Assume` that the environment is capable of running DinD pods, at least with the specified image version.

Reply:
Also failing locally when I run tests in #1275, but apparently not in CI.